docs/training-apis/api-core/distributed.txt

```diff
@@ -21,25 +21,41 @@ considerations are:
 #. Create a ``4_distributed.py`` training script by copying the ``3_hpsearch.py`` from
    :ref:`core-hpsearch`.
 
-#. In your ``if __name__ == "__main__"`` block, add ``multiprocessing`` code to launch one process
-   per slot.
+#. Add launcher logic to execute one worker subprocess per slot.
+
+   Typically, your distributed training framework provides a launcher, as Horovod provides
```

Comment:
I was never satisfied with this: maybe change it to "... **where** Horovod provides ..."

Reply:
I totally rewrote this section.  I also didn't like the order of how I introduced ideas.  Please re-read.